### PR TITLE
update read me with examples of trivial/non-trivial changes

### LINKED ISSUE #91: Make Governance section examples less vague
https://github.com/instructlab/dev-docs?tab=readme-ov-file#governance isn't very clear in it's examples - the wording here should be updated

### PR DESCRIPTION
**Pull request by lhawthorn**
And clarify when action must be taken by an oversight committee member. This PR is meant to address #91 

Closes #91

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,22 @@ The [InstructLab Oversight
...
 responsible for the contents of this repository.
 
-The rules for merging depend on the type of change in question and its scope of impact.
+The rules for merging depend on the type of change in question and its scope of impact. If you
+are unsure about the scope of impact for a change, i.e. if a change is trivial or non-trivial,
+please ping the Oversight Committee for help.
 
 * Trivial changes may be merged with 1 review from any InstructLab maintainer.
+  * Examples of trivial changes include minor wording adjustments or typo fixes in
+    documentation, changes to CI fixes, CI dependency updates, etc.
```
Comment (russellb) [inline README.md:22]:
```suggestion
    documentation, changes to CI fixes, CI dependency updates, etc.
```

File: README.md [hunk 2]
```diff
@@ -13,15 +13,22 @@ The [InstructLab Oversight
...
   controversial the change, the more broad the consensus should be required for
-  merging. The final approval and merge falls to a member of the Oversight Committee.
-  This final review is to ensure that adequate opportunity and attention has
-  been given by the affected parties.
+  merging. The final approval and merge (or action, e.g. deleting a repo)
+  falls to a member of the Oversight Committee. This final review is to ensure that
+  adequate opportunity and attention has been given by the affected parties.
+  * Examples of non-trivial changes include approving proposal for new repositories,
+    creation of new repositories, changes to organization level GitHub settings, archiving
+    or deleting repositories, etc.
```
Comment (russellb) [inline README.md:31]:
```suggestion
    or deleting repositories, design proposals, etc.
```

**Review by russellb — DISMISSED** [review]
Thank you!

- one spacing fix needed to make the linter happy. You should be able to just accept the suggested edit to fix it

- one suggested addition to the list of non-trivial examples

**Review by nathan-weinberg — APPROVED** [review]
Pending @russellb's comments this LGTM! Thank you for taking this on @lhawthorn!

**Review by hickeyma — APPROVED** [review]
Thanks @lhawthorn for adding the clarification.

**Comment by lhawthorn** [comment]
@mergifyio rebase

**Review by russellb — APPROVED** [review]

**Pull request merged**